CMakeLists.txt

```diff
@@ -167,22 +167,23 @@ endfunction()
 
 if ("x${CMAKE_CXX_COMPILER_ID}" STREQUAL "xMSVC" OR "x${CMAKE_CXX_COMPILER_FRONTEND_VARIANT}" STREQUAL "xMSVC")
   list(APPEND BLEND2D_PRIVATE_CFLAGS
-    -MP                      # [+] Multi-Process Compilation.
-    -GR-                     # [-] Runtime type information.
-    -GF                      # [+] Eliminate duplicate strings.
-    -Zc:__cplusplus          # [+] Conforming __cplusplus definition.
-    -Zc:inline               # [+] Remove unreferenced COMDAT.
-    -Zc:strictStrings        # [+] Strict const qualification of string literals.
-    -Zc:threadSafeInit-      # [-] Thread-safe statics.
-    -W4)                     # [+] Warning level 4.
+    -MP                           # [+] Multi-Process Compilation.
+    -GR-                          # [-] Runtime type information.
+    -GF                           # [+] Eliminate duplicate strings.
+    -Zc:__cplusplus               # [+] Conforming __cplusplus definition.
+    -Zc:inline                    # [+] Remove unreferenced COMDAT.
+    -Zc:strictStrings             # [+] Strict const qualification of string literals.
+    -Zc:threadSafeInit-           # [-] Thread-safe statics.
+    -Zc:arm64-aliased-neon-types- # [-] ARM64 aliased neon types
```

Comment:
I think this one option should be detected instead - it should use `blend2d_detect_cflags(BLEND2D_PRIVATE_CFLAGS, "-Zc:arm64-aliased-neon-types-")`